Editor/Window/RegisterComputeInput.cs

```diff
@@ -114,6 +118,49 @@ private async void OnRegisterComputeButtonClicked()
                         Strings.ViewLogsStatusBoxUrlTextButton);
                 }
             }
+            
+            UpdateGUI();
+        }
+
+        private async void CheckCompute()
+        {
+            if (_computeState is ComputeStatus.Registered)
```

Comment:
If the fleet changed, the previous compute state doesn't matter at all. This check seems redundant. We should query the service to see if the compute the stateManager thinks is registered still exists or not and set the _computeState accordingly.

Reply:
Removed and added a setting of registered if it finds that it does have that compute.